flocker/docs/_version.py

```diff
@@ -163,3 +169,37 @@ def is_pre_release(version):
             and parsed_version.weekly_release is None
             and parsed_version.commit_count is None
             and parsed_version.dirty is None)
+
+
+def get_pre_release(version):
```

Comment:
`_version` is private and so I was thinking that the class was not directly exposed for a reason - perhaps because we will be changing our versioning to fit with PEP440. I was concerned about exposing more of an interface than was necessary for this branch.